site/tsconfig.json

```diff
@@ -3,6 +3,8 @@
     "compilerOptions": {
         "strict": true,
         "allowJs": true,
-        "noEmit": true
+        "noEmit": true,
+        "incremental": true,
+        "outDir": "incremental_build"
```

Comment:
Please add this folder to gitignore (`/site/incremental_build/`) since we don't need that checked in for someone who runs tsc outside of the install script.

Reply:
Thanks for catching that